### ISSUE TITLE
Report violation if default parameter value is not documented

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#519]
We should issue a warning if the default value of an optional parameter is not documented (except for the `params` parameter).
MiKo_2076

Following code should issue a warning:
```c#
/// <summary>Does something</summary>
/// <param name="value">Some value</param>
public void DoSomething(bool value = false)
{ }
```

**Comment by RalfKoban** [comment]
CodeFix provider has been provided in 1e957c1c6cf69b70b113124d737c5df2186ed28c
